### ISSUE TITLE
Broken release `v0.3.0-3`

### ISSUE THREAD

**Issue by fatih** [kubernetes-csi/csi-test#137]
Hi,

I was using `v0.3.0-2` for https://github.com/digitalocean/csi-digitalocean with the CSI Spec `v0.3.0` tagged release (see: https://github.com/container-storage-interface/spec/releases/tag/v0.3.0).

I decided to upgrade to `v0.3.0-3` as it was including the latest changes.  However when I upgrade it, my test are failing due the following error:

```
# github.com/kubernetes-csi/csi-test/pkg/sanity
../../go/pkg/mod/github.com/kubernetes-csi/csi-test@v0.3.0-3/pkg/sanity/node.go:122:10: undefined: csi.NodeServiceCapability_RPC_GET_VOLUME_STATS
```

When I check the CSI `v0.3.0` version, there is no such identifier. This was added later to CSI master and is part of the currently released `v1.0.0-rc1` release (https://github.com/container-storage-interface/spec/releases/tag/v1.0.0-rc1). 

When I update my CSI spec to `v1.0.0-rc1` from `v0.3.0`, it still fails, albeit with a different error:

```
go: import "github.com/digitalocean/csi-digitalocean/driver" ->
        test ->
        import "github.com/kubernetes-csi/csi-test/pkg/sanity" ->
        import "github.com/container-storage-interface/spec/lib/go/csi/v0": looping trying to add package
```

The reason is, CSI `v1.0.0-rc1` get rid of the `/v0` suffix from their import path, but the CSI-sanity tests still uses the `/v0` suffix, example: https://github.com/kubernetes-csi/csi-test/blob/master/driver/driver.go#L32

Because of this, the `v0.3.0-3` is not compatible with either `v0.3.0` or `v1.0.0-rc1` of the CSI spec.

I think a new `v0.3.0-4` version should be released that removes the `csi.NodeServiceCapability_RPC_GET_VOLUME_STATS` identifier because it's not part of `v0.3.0`, or we should release a csi-test `v1.0.0-rc1` release that is compatible with CSI spec `v1.0.0-rc1` release that was released today.

**Comment by lpabon** [comment]
Thank you for bringing this up @fatih . @pohly figured out that the CSI spec had been changed in our repo.  Also, the importing issue, is new to me. I'll have to investigate.

**Comment by lpabon** [comment]
We have released 1.0.0 and are working on a new 0.3.0-4 (#140) release

**Comment by pohly** [comment]
0.3.0-4 has been tagged.

Note that "dep" does not pick it up unless explicitly chosen with `version = "0.3.0-4"` because of the -4 suffix. Future stable releases should be using just x.y.z as version numbers.

/close
